README.md

```diff
@@ -295,6 +295,37 @@ func main() {
 }
 ```
 
+## Emitting a schema.json
+
+It may be useful to emit a `schema.json` file containing the results of the
+GraphQL introspection query schema for compatibility with other GraphQL tooling.
+Alongside the above example, here is a small program for registering our schema and
+writing the JSON output to stdout.
+
+```go
+// schema_generator.go
```

Comment:
I wonder if we could make this easier. Could we make running the server emit a schema? Could we add a function that takes a schema and writes a file to the filesystem, and stick that in the examples above?

Reply:
The former seems a little too magical. The latter seems nice, but I want to document how this is currently possible for now.